src/plugins/workspace/common/constants.ts

```diff
@@ -3,6 +3,13 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+export const WORKSPACE_CREATE_APP_ID = 'workspace_create';
+export const WORKSPACE_LIST_APP_ID = 'workspace_list';
+export const WORKSPACE_UPDATE_APP_ID = 'workspace_update';
+export const WORKSPACE_OVERVIEW_APP_ID = 'workspace_overview';
+// These features will be checked and disabled in checkbox on default.
+export const DEFAULT_CHECKED_FEATURES_IDS = [WORKSPACE_UPDATE_APP_ID, WORKSPACE_OVERVIEW_APP_ID];
```

Comment:
DEFAULT_CHECKED_FEATURES_IDS, I'm assuming that you want to mention here by `selected`, like DEFAULT_SELECTED_FEATURES_IDS, right? Thanks

Reply:
I think we can rename to it.